NitroxClient/Communication/Packets/Processors/StoryEventProcessor.cs

```diff
@@ -0,0 +1,67 @@
+﻿using NitroxClient.Communication.Abstract;
+using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxModel.Core;
+using NitroxModel.Helper;
+using NitroxModel.Packets;
+using Story;
+
+namespace NitroxClient.Communication.Packets.Processors
+{
+    public class StoryEventProcessor : ClientPacketProcessor<StoryEventSend>
+    {
+        public override void Process(StoryEventSend packet)
+        {
+            switch (packet.StoryEventType)
+            {
+                case StoryEventType.PDA:
+                case StoryEventType.RADIO:
+                case StoryEventType.ENCYCLOPEDIA:
+                case StoryEventType.STORY:
+                    using (NitroxServiceLocator.LocateService<IPacketSender>().Suppress<StoryEventSend>())
```

Comment:
Add constructor to set `IPacketSender` as local field. `NitroxServiceLocator` is slow and should not be used if possible (because it's an anti pattern).